src/utils/commandLine.ts

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+import blessed = require('blessed');
+
+export function createMessageBox(view: blessed.widget.Screen): MessageBoxMethods {
+	let message: blessed.widget.Box = null;
+	const messageText: blessed.widget.Box[] = [];
+
+	(<MessageBoxMethods> add).add = add;
+	(<MessageBoxMethods> add).edit = edit;
+	(<MessageBoxMethods> add).remove = remove;
```

Comment:
These all say `<MessageBoxMethods> add`, pretty sure you mean `edit` and `remove` respectively.